Sparkle/SUUIBasedUpdateDriver.m

```diff
@@ -162,6 +163,9 @@ - (void)updateAlertFinishedWithChoice:(SUUpdateAlertChoice)choice forItem:(SUApp
             break;
 
         case SUSkipThisVersionChoice:
+            if ([[updater delegate] respondsToSelector:@selector(userDidSkipThisVersion:)]) { 
+                [[updater delegate] updater:self.updater userDidSkipThisVersion:self.updateItem]; 
```

Comment:
How did this pass review? respondsToSelector is checked with one selector, while another is called inside. The inside selector is unknown, causing compile error.

https://github.com/sparkle-project/Sparkle/issues/1455